src/partials/_calculator.scss

```diff
@@ -120,26 +165,62 @@ h1 {
 }
 
 .calculator__input__spec {
-  margin: 7px;
   background-color: #ffe62e;
   box-shadow: 0 0 20px deeppink;
-  flex-basis: 20%;
+  position: relative;
+  transition: all 5s ease-in-out;
+  width: 100%;
+  height: 100%;
   font-size: 3rem;
   border-radius: 20px;
   outline: none;
 }
 
+// .calculator__inputs__spec.active {
+//   position: relative;
+//   left: 0px;
+//   bottom: 0px;
+// }
+
 .calculator__input__result {
+  font-size: 3rem;
   background-color: #ffe62e;
   box-shadow: 0 0 20px deeppink;
-  flex-basis: 20%;
-  font-size: 3rem;
   border-radius: 20px;
+
+  outline: none;
+  position: relative;
+  transition: all 5s ease-in-out;
+  top: -500px;
+  left: -500px;
+  width: 100%;
+  height: 100%;
+}
+
+// .calculator__input__result.active {
+//   position: relative;
+//   right: 0px;
+//   bottom: 0px;
+// }
+
+.calculation {
+  font-size: 2rem;
+  background-color: springgreen;
+}
+
+.active {
+  position: relative;
+  right: 0px !important;
```

Comment:
`!important` is not a good practice. can you think about a different solution?

Reply:
That's right and i'm not happy with this solution either. But my css properties got overwritten in a way i don't want it, so like the comment above, i couldn't find another solution yet.